app-main/public/css/style_404.css

```diff
@@ -0,0 +1,40 @@
+body {
+  margin: 0;
+  padding: 0;
+  font-family: system-ui, -apple-system, BlinkMacSystemFont, Segoe UI, Roboto,
+    Oxygen, Ubuntu, Cantarell, Fira Sans, Droid Sans, Helvetica Neue;
+  letter-spacing: 0.02em;
+  background-color: #339989;
+}
+
+#message {
```

Comment:
In general it's best to stick with class rather than ID selectors. You get more wiggle room in terms for specificity with a class. No need to change but thought I'd mention it 😄

Reply:
Oh! I didn't know that-thank you!